kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -175,6 +183,11 @@ public void put(Collection<SinkRecord> records) {
     Map<PartitionedTableId, TableWriterBuilder> tableWriterBuilders = new HashMap<>();
 
     for (SinkRecord record : records) {
+      if (sinkRecordFilter.getConfigType() != null) {
```

Comment:
we should remove this condition and incorporate this into the shouldSkip method so the caller doesn't have to understand the implementation details of the SinkRecordFilter.